greynoise_connector.py

```diff
@@ -32,6 +43,7 @@ def __init__(self):
         self._session = None
         self._app_version = None
         self._api_key = None
+        self._integration_name = "splunk-soar-v2.5.0"
```

Comment:
@superducktoes : looks like the desire here is for this to be v2.2.0.  We should update that, and also this value should probably be in the const file.  Can you please push an update?

Reply:
@bradchiappetta just updated